specification/communication/resource-manager/Microsoft.Communication/stable/2020-08-20/CommunicationService.json

```diff
@@ -0,0 +1,1118 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure Communication Services Management API",
+    "version": "2020-08-20",
```

Comment:
As you are already used 2020-08-20-preview, recommend to bump up the GA version to avoid confusion.
To easily identify the delta changes from previous preview api version, the recommendation is to copy the existing version as is into new directory structure for first commit and then push new changes in subsequent commits. Please raise a new PR as suggested for faster review.

Reply:
We asked the team earlier for the GA best practices and this is what was recommended, upgrade preview version to stable by removing the -preview suffix which actually tells the story that we worked on it before to reach stable version, and that made sense. Changing the api-version now will incur more changes in our backend service and arm manifest, and I would prefer not doing that so close to GA, the team is busy with stabilization for GA. Thanks for understanding

Reply:
@RamyasreeChakka Is this issue blocking the ARM signoff of this review? We don't intend to change this. Let's discuss so that we can get this signed off.